bin/Analysis.cpp

```diff
@@ -151,6 +155,9 @@ void Analysis::createHistograms() {
 			<< endl;
 	lastNumberOfHistograms = numberOfHistograms;
 
+	pseudoTopAnalyser_->createTrees();
```

Comment:
Quick question:
Are these trees created for every datatype and not just for TTJetty ones?